autopub/github_contributor.py

```diff
@@ -0,0 +1,85 @@
+import subprocess
+import os
```

Comment:
Total nitpick, but perhaps swap the position of these two imports, so they are in alphabetical order?